apps/editing-toolkit/editing-toolkit-plugin/wpcom-site-editor/paragraph-block/placeholder.jsx

```diff
@@ -0,0 +1,22 @@
+import { addFilter } from '@wordpress/hooks';
+
+addFilter(
+	'blocks.registerBlockType',
+	'full-site-editing/wpcom-site-editor/paragraph-block-placeholder',
+	( settings, name ) => {
+		if ( name !== 'core/paragraph' ) {
```

Comment:
Maybe it's better to just bypass changing the override until the translation is ready.

```suggestion
		if ( name !== 'core/paragraph' || ! window.wpcomSiteEditorParagraphPlaceholder ) {
```